tests/long-opts/conv2d-nchw/convolution_nchw.src.mlir

```diff
@@ -1,10 +0,0 @@
-// VERIFY
```

Comment:
Is there a reason why this test has been removed?

Reply:
oops. forgot to add file after renaming dir